fapi-client/src/main/scala/com/gu/facia/api/utils/ResolvedMetaData.scala

```diff
@@ -18,17 +22,21 @@ object ResolvedMetaData {
   def isCommentForContent(content: Content): Boolean =
     content.tags.exists(_.id == Comment)
 
-  def isVideoForContent(content: Content): Boolean =
-    content.tags.exists(_.id == Video)
+  def isVideoElementForContent(content: Content): Boolean =
+    content.tags.exists(_.id == Video) && content.elements.exists(_.exists(_.`type` == ElementType.Video))
```

Comment:
Why are we now adding this extra restriction?
It's up to the client to request the data, but I don't think we should set defaults based on whether the client has requested it or not. Rather, I think we should always set the default and it's up to the client to render it appropriately.
Although maybe we could change how it works and do it like this now instead?